src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -469,7 +477,8 @@ def bulk_set(self, flags):
                                     'to.')
                  },
                  responses={200: EXPORT_INDIV_FLAGS_SCHEMA})
-    def set_individual(self, flags, ids):
+    @typechecked
+    def set_individual(self, flags: IndividualFlags, ids: List[int]):
```

Comment:
if this API is only consumed by ourselves, i.e. it's an internal API. do we still need to install `typeguard` in production environment, or we just need to install it in our testing environment, when running teuthology based test and `make check` tests?

Reply:
@tchaikov The idea of this improvement is to be on production code. Duck-typing is great most of the time, but if you're expecting a list of labels for a host, but the API user sends just a single label, both input args (`list` and `str`) are iterable, but in one case you get a host labelled liked `['rgw', 'mds']`, while in the other `['r', 'g', 'w']`... So we need to actively enforce typecheck in methods. We could populate with `asserts` and maintain the assertions in sync with the type hints... but that's what `typeguard` does for us 🥳